### PR TITLE
Add more scrolling dartdocs

### PR DESCRIPTION
**Pull request by abarth**
Also, add a few other random dartdocs that I encountered along the way.

### CODE REVIEW

**Comment by abarth** [comment]
@Hixie @HansMuller

File: packages/flutter/lib/src/widgets/banner.dart [hunk 1]
```diff
@@ -60,8 +60,10 @@ class BannerPainter extends CustomPainter {
   /// Where to show the banner (e.g., the upper right corder).
   final BannerLocation location;
 
+  /// The color to paint behind the [message].
   final Color color;
 
+  /// The text style to use for the [message].
```
Comment (Hixie) [inline packages/flutter/lib/src/widgets/banner.dart]: Say what it defaults to.

File: packages/flutter/lib/src/widgets/banner.dart [hunk 2]
```diff
@@ -60,8 +60,10 @@ class BannerPainter extends CustomPainter {
   /// Where to show the banner (e.g., the upper right corder).
   final BannerLocation location;
 
+  /// The color to paint behind the [message].
```
Comment (Hixie) [inline packages/flutter/lib/src/widgets/banner.dart]: Say what it defaults to.

**Comment by Hixie** [comment]
LGTM

**Pull request merged**